tests/integration/conftest.py

```diff
@@ -32,6 +33,16 @@ def run_action(ops_test: OpsTest) -> Callable[[str, str], Awaitable[Any]]:
     """Create a async function to run action and return results."""
 
     async def _run_action(application_name: str, action_name: str, **params):
+        """Run a specified action.
+
+        Args:
+            application_name: Name the application is deployed with.
+            action_name: Name of the action to be executed.
+            params: Dictionary with parameters.
```

Comment:
Perhaps indicate what they are used for? Would that help?

Reply:
There is not much to describe on the documentation about the [params](https://pythonlibjuju.readthedocs.io/en/latest/api/juju.unit.html#juju.unit.Unit.run_action), but will clarify that they're action parameters.